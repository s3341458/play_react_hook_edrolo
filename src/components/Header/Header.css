.header {
  border: 1px solid #ddd;
}
