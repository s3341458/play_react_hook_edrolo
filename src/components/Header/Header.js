import React from "react";
import style from './Header.css';

const Logo = () => (
	<a href="/">
		<img src="/img/edrolo-logo.png" />
	</a>
);

const Header = () => {
	return (
		<header className={style.header}>
			<Logo />
			<nav>
				<ul>
					<li>
						<a href="/">Home</a>
					</li>
					<li>
						<a href="/about">About</a>
					</li>
				</ul>
			</nav>
		</header>
	);
};

export default Header;
