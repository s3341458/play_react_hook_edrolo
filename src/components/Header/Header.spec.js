/* eslint-env mocha */
import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from './Header';

describe('<Heading />', () => {
  it('should render a Header', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
})
