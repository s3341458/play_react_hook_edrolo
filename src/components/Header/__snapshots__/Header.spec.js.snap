// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render a Header 1`] = `
<header
  className="header"
>
  <Logo />
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about"
        >
          About
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
